org.eclipse.lsp4e/plugin.xml

```diff
@@ -329,6 +340,52 @@
             </visibleWhen>
          </command>
       </menuContribution>
+      <menuContribution
+            locationURI="popup:#TextEditorContext?before=org.eclipse.ui.genericeditor.findReferences">
+         <command
+               commandId="org.eclipse.lsp4e.typeHierarchy">
+            <visibleWhen
```

Comment:
If you properly configure `enabledWhen` on the handler, then it can be replaced with `visibleWhen checkEnabled="true"`